INSTALL.md

```diff
@@ -38,10 +38,22 @@ Dangerzone is available for:
   ```
 </details>
 
-Add our repository following [these instructions](https://packagecloud.io/firstlookmedia/code/install#manual-deb), or by running this script:
-
-```
-curl -s https://packagecloud.io/install/repositories/firstlookmedia/code/script.deb.sh | sudo bash
+Add our repository following these instructions:
+
+```sh
+sudo apt-get update
+sudo apt-get install -y curl
```

Comment:
Damn. So Debian has `curl` by default but not `wget` and Ubuntu is the other way around :/